torch/optim/_multi_tensor/adamax.py

```diff
@@ -96,7 +98,7 @@ def step(self, closure=None):
                     exp_inf.unsqueeze(0),
                     grad.abs().add_(eps).unsqueeze_(0)
                 ], 0)
-                torch.max(norm_buf, 0, keepdim=False, out=(exp_inf, exp_inf.new().long()))
+                torch.amax(norm_buf, 0, keepdim=False, out=exp_inf)
```

Comment:
Are there any behavior changes here?

Reply:
No behavior changes, this change is picked from regular optimizer. Before we had `amax`, we were forced to use `max` function that returned indices, and created a dummy tensor to hold indices, that's not necessary now.